iphone/cli/commands/_build.js

```diff
@@ -2114,23 +2114,20 @@ iOSBuilder.prototype.validate = function validate(logger, config, cli) {
 
 			function determineMinIosVer() {
 				// figure out the min-ios-ver that this app is going to support
-				let defaultMinIosSdk = this.packageJson.minIosVersion;
-
-				if (version.gte(this.iosSdkVersion, '10.0') && version.lt(defaultMinIosSdk, '9.0')) {
-					defaultMinIosSdk = '9.0';
-				}
-
-				this.minIosVer = this.tiapp.ios['min-ios-ver'] || defaultMinIosSdk;
-
-				if (version.gte(this.iosSdkVersion, '10.0') && version.lt(this.minIosVer, '9.0')) {
-					logger.warn(__('The %s of the iOS section in the tiapp.xml is lower than the recommended minimum iOS version %s', 'min-ios-ver', '9.0'));
-					logger.warn(__('Consider bumping the %s to at least %s', 'min-ios-ver', '9.0'));
-				} else if (version.gte(this.iosSdkVersion, '6.0') && version.lt(this.minIosVer, defaultMinIosSdk)) {
-					logger.info(__('Building for iOS %s; using %s as minimum iOS version', version.format(this.iosSdkVersion, 2).cyan, defaultMinIosSdk.cyan));
-					this.minIosVer = defaultMinIosSdk;
-				} else if (version.lt(this.minIosVer, defaultMinIosSdk)) {
-					logger.info(__('The %s of the iOS section in the tiapp.xml is lower than minimum supported version: Using %s as minimum', 'min-ios-ver'.cyan, version.format(defaultMinIosSdk, 2).cyan));
-					this.minIosVer = defaultMinIosSdk;
+				let defaultMinIosVersion = this.packageJson.minIosVersion;
+
+				this.minIosVer = this.tiapp.ios['min-ios-ver'] || defaultMinIosVersion;
+
+				if (version.gte(this.iosSdkVersion, '10.0') && version.lt(this.minIosVer, '10.0')) {
```

Comment:
I think maybe this check and the check on 2125 _used_ to be useful (this line looks to have been to force people down the right path #8198) but now our minimum is 10 maybe it's redundant?

`this.iosSdkVersion >= 10` and `this.iosSdkVersion >= 6` will always be true nowadays, and `this.minIosVer < 10` being true seems to duplicate the check on 2128 where we force the minimum to 10 if someone sets it to 9??